### PR TITLE
more strict flake8. Fixed typos.

### PR DESCRIPTION
**Pull request by HerrMuellerluedenscheid**
I disabled `flake8` checks on `E722` (do not use bare 'except') for now. But should be scrutinized and fixed. See #39.

### CODE REVIEW

**Comment by FlorianLudwig** [comment]
Do you have a hint how I can view an actual diff between this branch and master?

The diff shows all changes and not a diff to master. It does contains my changes from the refactoring.

:thinking:

**Comment by FlorianLudwig** [comment]
Okay, maybe the diff looks like this because this was a merge into this branch from master and not a rebase, could you try rebasing @HerrMuellerluedenscheid ?

**Comment by HerrMuellerluedenscheid** [comment]
> Okay, maybe the diff looks like this because this was a merge into this branch from master and not a rebase, could you try rebasing @HerrMuellerluedenscheid ?

Exactly. Hastily, quickly before next meeting rebase-fu**-up :blush: I'm going to fix this after lunch. I'll switch to non-draft when it's ready and send you a heads up.

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -31,7 +31,6 @@ jobs:
     - name: Lint with flake8 and black
       run: |
         flake8 . --count --statistics --show-source --ignore=E501,W503,W605,E722
-        flake8 . --count --statistics --exit-zero --max-complexity=10 --ignore=E501,W503
```
Comment (HerrMuellerluedenscheid) [inline .github/workflows/python-package.yml]: The output of the CI looks misleading when e.g. black fails as there are plenty of errors being thrown at the moment from this line. Should be fixed before enabling complexity checks and others.

**Review by FlorianLudwig — APPROVED** [review]
LGTM

**Pull request merged**